### ISSUE TITLE
Pass precision matrix to multivariate gaussian

### ISSUE THREAD

**Issue by jucor** [deepmind/torch-distributions#6]
_Issue by **[jucor](https://github.com/jucor)** from Friday Nov 01, 2013 at 18:30 GMT_
_Originally opened as https://github.com/jucor/torch-randomkit/issues/6_

---

Suggested by Shakir.

**Comment by jucor** [comment]
Ping @dpfau : we agreed on Friday that this was probably indeed a good idea :) I just realize I made this note to myself above. Any thoughts on it? Regardless, since many packages offer this, we should implement it.

**Comment by dpfau** [comment]
Agreed it would be nice - more control over when you need to do a matrix inversion is never a bad thing.
